src/test/java/christmas/domain/plan/BadgeTest.java

```diff
@@ -0,0 +1,28 @@
+package christmas.domain.plan;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class BadgeTest {
+
+    @DisplayName("[SUCCESS] 혜택 금액에 알맞은 배지를 반환한다")
+    @ParameterizedTest
```

Comment:
아래와 같이 각각의 파라미터 테스트 역시 이름을 부여해 줄 수 있습니다!
```suggestion
    @ParameterizedTest(name = "총 구매 금액이 {0}원이면 {1} 뱃지가 반환된다.")
```